engine/swarm/services.md

```diff
@@ -603,6 +641,14 @@ $ docker service create \
   redis:3.0.6
 ```
 
+The syntax is a little different from the previous example. It is a key-value
+pair where the key refers to a placement strategy (currently, `spread` is the
+only supported strategy), and the value is a string using dotted notation, where
+each field refers to a category of metadata. In this case, the service will be
+spread across nodes with the node label `region` set to `east`. If you left off
+the last field, the service would be spread across nodes which had `region` set
+to any value.
```

Comment:
This last part doesn't seem right. Where does `east` enter the picture? What is "the last field" referring to?